thoth/user_api/api_v1.py

```diff
@@ -535,10 +535,10 @@ def schedule_kebechet(body: dict):
     headers = connexion.request.headers
     if "X-GitHub-Event" in headers:
         service = "github"
-        url = body["repository"]["url"]
+        url = body["repository"]["html-url"]
```

Comment:
@harshad16 is it html-url or html_url ?

Reply:
yes it is html_url as per https://developer.github.com/v3/activity/events/types/#webhook-payload-example-34
my bad, missed it while reviewing.

Reply:
not your bad!